metadata/overlays/istio/virtual-service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ui
+spec:
+  gateways:
+  - kubeflow-gateway
+  hosts:
+  - '*'
+  http:
+  - match:
+    - uri:
+        prefix: /metadata/
+    rewrite:
+      uri: /metadata/
```

Comment:
Do we have common path prefix `/metadata/` on metadata UI? Or rewrite should be `/`?